master/docs/conf.py

```diff
@@ -21,7 +21,18 @@
 sys.path.append(os.path.abspath('.'))
 
 # -- General configuration -----------------------------------------------------
-
+try:
+    import sphinxcontrib.blockdiag
```

Comment:
add `  # noqa` here to ignore flake8 linting errors (with two spaces before `#` sign):

```
    import sphinxcontrib.blockdiag  # noqa
```